src/EditorFeatures/Core/Implementation/Classification/ClassificationTypeDefinitions.cs

```diff
@@ -33,6 +33,12 @@ internal sealed class ClassificationTypeDefinitions
         [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
         internal readonly ClassificationTypeDefinition StringEscapeCharacterTypeDefinition;
         #endregion
+        #region Keyword - Control
+        [Export]
+        [Name(ClassificationTypeNames.ControlKeyword)]
+        [BaseDefinition(PredefinedClassificationTypeNames.Keyword)]
```

Comment:
interesting. Does this have any sort of user observable impact?

Reply:
Control keywords will inherit the styling of Keywords and appear as keywords in the absence of styling configuration. I'll add a comment. =)